lib/still.ex

```diff
@@ -1,3 +1,13 @@
 defmodule Still do
   @moduledoc false
+
+  @doc """
+  Registers a callback to be called synchronously after the compilation.
+  """
+  defdelegate on_compile(fun), to: Still.Compiler.Compile
```

Comment:
I know we already discussed this, but should we find another name? Since it calls _after_ the compilation.

Reply:
Yes, `after_compile`  👍